core/src/components/modal/animations/sheet.ts

```diff
@@ -40,10 +46,11 @@ export const createSheetLeaveAnimation = (opts: ModalAnimationOptions) => {
    * is defined, so we need to account for that offset by figuring out
    * what the current backdrop value should be.
    */
-  const backdropValue = `calc(var(--backdrop-opacity) * ${getBackdropValueForSheet(
-    currentBreakpoint!,
-    backdropBreakpoint!
-  )})`;
+  const backdropValue =
+    opts.theme === 'ionic'
```

Comment:
Let's not using theme to check. I would use `opts.backdropValue`. Read the comment below regarding this.